dashboard/app/views/admin_users/permissions_form.html.haml

```diff
@@ -1,28 +1,77 @@
-%h1 Permissions Form
+%h1 Manage User Permissions
 
-%h2 Grant Permission
+#application-container
+  = show_flashes.html_safe
 
-=form_tag(grant_permission_path, method: :post) do
-  email
-  =text_field_tag 'email', ''
-  %br
-
-  permission
-  =text_field_tag 'permission', ''
-  %br
-
-  =submit_tag 'Grant Permission'
-
-%h2 Revoke All Permissions
-
-=form_tag(revoke_all_permissions_path, method: :post) do
-  email
-  =text_field_tag 'email', ''
-  %br
-
-  = submit_tag 'Revoke All Permissions'
-
--# TODO(asher).
-%h2 Revoke Permission
-
-Coming soon.
+%p Enter an Email or User ID to view and edit a User's permissions
+= form_tag url_for(action: 'permissions_form'), method: 'get', class: 'form-inline', enforce_utf8: false do
+  = text_field_tag :search_term, params[:search_term], class: 'form-control'
+  %button.btn{type: 'submit'}
+    %i.fa.fa-search
+-if @user
+  %table.table.table-hover.table-condensed.table-auto-width
+    %thead
+      %th ID
+      %th Email
+      %th Name
+      %th User Type
+      %th Last Sign In
```

Comment:
Do you mean "Last Sign In" or "Current Sign In"? Note that `User.current_sign_in_at` is the most recent sign-in timestamp and `User.last_sign_in_at` is the second-most recent sign-in timestamp (unless there has only been one sign-in, in which case they are equal).